libafl/src/stages/concolic.rs

```diff
@@ -58,6 +58,17 @@ where
             ));
         };
 
+        <TracingStage<EM, TE, Z> as Stage<E, EM, Z>>::Progress::initialize_progress(
```

Comment:
W H A T

I'm surprised that's even legal rust, is there any way to write this in a simpler way?

Reply:
Bit of an inelegance that Rust has... sometimes it needs the specific type that an associated type is being pulled from but here it should be deducible. I'll find a better way to write this.

Reply:
Resolved the same way as above.